docs/docs/alephium/handling_alph.md

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
A third point akin to 

## ALPH as token in entrypoints

with short mention on how to get the contract id, how it should be sent through the attoAlphAmount field instead of tokens etc
this explanation partially isn't our responsibility, but better to include it since there are no other good resources to point to for that, and users might first and foremost browse our docs for hints